src/main/java/net/teamterminus/machineessentials/energy/electric/api/VoltageTier.java

```diff
@@ -0,0 +1,32 @@
+package net.teamterminus.machineessentials.energy.electric.api;
+
+import net.modificationstation.stationapi.api.util.Formatting;
+import net.teamterminus.machineessentials.MachineEssentials;
+
+import java.util.List;
+
+/**
+ * Voltage tiers from ULV (8V) to MAX (65535V), with their colors and names.
+ */
+public record VoltageTier(String voltageName, int minVoltage, int maxVoltage, Formatting textColor, int color) {
```

Comment:
Maybe replace voltage names from strings to lang files? (So it can be translated)

Reply:
Makes sense